src/flamenco/txn/fd_txn_generate.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * Provides utility methods to create txn templates for
+ * pre-staging, as well as a mechanism to build out an 
+ * entire transaction with instructions.
+*/
```

Comment:
Firedancer comments start with `/*` and don't have `*`s on subsequent lines. The ending `*/` can be on the same line as the text.